addons/point_of_sale/models/stock_picking.py

```diff
@@ -191,24 +191,26 @@ def _load_pos_data_fields(self, config):
         return ['id', 'use_create_lots', 'use_existing_lots']
 
 
-class ProcurementGroup(models.Model):
-    _inherit = 'procurement.group'
+class StockReference(models.Model):
+    _inherit = 'stock.reference'
 
-    pos_order_id = fields.Many2one('pos.order', 'POS Order')
+    pos_order_ids = fields.Many2many('pos.order', 'POS Order')
```

Comment:
Isn't this the same in the file below?
addons/point_of_sale/models/stock_reference.py

Reply:
@aj-fuentes You are right. thanks